rfcs/rfc-template.md

```diff
@@ -41,6 +41,14 @@
 
 *Explain the design in sufficient detail such that the implementation and (if appropriate) the interaction with existing DCP software are both reasonably clear.*
 
+**ADDITION BEGINS**
+
+### Timeline [optional]
```

Comment:
Let me take one step back. The problem I was trying to solve here was to curb the number of non-informational RFCs that are approved but never get implemented. This also influences the review process because approvers look at the RFC as "yeah this seems reasonable someday" which is not a great way to review non-informational RFCs.

That's what I was trying to accomplish by adding this optional Timeline section. We could either ensure that when a Spike is created, that time is watermarked in future quarters for the implementation of resulting design from the Spike which would ensure that this section doesn't deviate from the quarterly roadmap. Or part of the quarterly roadmap refresh should account for any RFCs' timelines that have been proposed here.

Note: for informational RFCs, this section does not make sense to fill out, hence optional.

Do you have thoughts about the above/any alternative ideas?

Reply:
I recommend reviewing [Refreshing the Product Roadmap](https://github.com/HumanCellAtlas/dcp-community/blob/daf85fa4d7fb4b9d9805cf559b4eda59f18c9686/rfcs/text/0000-roadmaps%2Bplanning.md#refreshing-the-product-roadmap). There is no quarterly roadmap. The long-term roadmap is simply refreshed on a quarterly basis to include technical objectives and any emerging conditions. Currently, the roadmap does not have a timeline either - that's determined by the quarterly engineering plan what takes a slice of the roadmap priorities based on "fit".

In general, I would expect most RFC investment to be focused on product roadmap priorities rather than _yeah this seems reasonable someday_?

Reply:
Based on our conversation from yesterday, removing the Timeline section.